superdesk/eve_backend.py

```diff
@@ -22,6 +22,35 @@
 from eve.methods.common import resolve_document_etag
 from elasticsearch.exceptions import RequestError, NotFoundError
 from superdesk.errors import SuperdeskApiError
+from superdesk.notification import push_notification
+
+
+SYSTEM_KEYS = set([
```

Comment:
you might want to also add `_links` and `_id`

Reply:
these are not stored in db so it won't make it there (`_id` is stored but mongo won't accept it in update payload)